src/ensembl/src/content/app/entity-viewer/gene-view/components/transcripts-table/TranscriptsTable.tsx

```diff
@@ -0,0 +1,109 @@
+import React, { useState } from 'react';
+
+import MainContent from './main-content/MainContent';
+import SecondaryContent from './secondary-content/SecondaryContent';
+import { ImageButton } from 'src/shared/components/image-button/ImageButton';
+
+import { ReactComponent as chevronLeftIcon } from 'static/img/shared/chevron-left.svg';
+import { ReactComponent as chevronRightIcon } from 'static/img/shared/chevron-right.svg';
+
+import { Status } from 'src/shared/types/status';
+
+import styles from './TranscriptsTable.scss';
+
+const TranscriptsTable = () => {
+  const [paginationNumber, setPaginationNumber] = useState(1);
```

Comment:
`paginationNumber` → `page`
`setPaginationNumber` →`setPage`

Reply:
It's not exactly a page and that's why I went with pagination. But in hindsight that is still telling us about a page. How about panel instead?

Reply:
I am ok with panels if you prefer them to pages.

Reply:
:heavy_check_mark: